src/patternfly/components/Menu/menu-item.hbs

```diff
@@ -23,5 +29,20 @@
       aria-expanded="false"
     {{/if}}
   {{/if}}>
-  {{> @partial-block}}
-</{{# if menu-item--IsLink}}a{{else if menu-item--type}}{{menu-item--type}}{{else}}button{{/if}}>
+  {{#if menu-list-item--IsCheckbox}}
+    {{#> menu-item-main}}
+      {{#> menu-item-check}}
+        {{#> check check--type="span" check--IsDisabled=menu-list-item--IsDisabled}}
```

Comment:
We should probably use the standalone modifier, though it seems to throw off the alignment a bit... wdyt?

```suggestion
        {{#> check check--type="span" check--modifier="pf-m-standalone" check--IsDisabled=menu-list-item--IsDisabled}}
```